deps/quickjs/CMakeLists.txt

```diff
@@ -52,3 +52,9 @@ target_include_directories(qjs PRIVATE src)
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
     target_link_libraries(qjs atomic)
 endif()
+
+if(MINGW)
+    target_compile_definitions(qjs PRIVATE
+        CONFIG_PRINTF_RNDN
```

Comment:
@saghul 
I find quickjs has definde `CONFIG_PRINTF_RNDN`, but my windows not has curl lib, so this code is not testing